<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>todo_window</class>
 <widget class="QMainWindow" name="todo_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>569</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>To-do List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(165, 166, 151);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="todo_list">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 1, 38);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TO-DO LIST</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="done_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="rights_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(13, 1, 38);</string>
    </property>
    <property name="text">
     <string>FEITO POR NICOLAS KLEIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="todo_item">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>450</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
